code/script.js

```diff
@@ -0,0 +1,122 @@
+                    <div class= project-card>
+                        <a class= "project-names-with-pr" href="${
+                          repo.html_url
+                        }" target="_blank">${repo.name}</a>
+                        <p> <a href="${
+                          repo.homepage
+                        }" target="_blank" class="view-live" >View it Live ▶ </a> </p>
+                        <span class="repo-language"></span>
+                        <span class="language-name"> ${repo.language}</span>
+                        <p>Default branch: ${repo.default_branch} </p> 
+                        <p> Recent push: ${new Date(
+                          repo.pushed_at
+                        ).toDateString()}</p>
+                        <p class= "commits-info"id="commit-${repo.name}"> </p>
+                    </div> 
+                     `
+
+        // display the most used language for each project and change the color depending on the language
+        const repoLanguage = document.querySelectorAll(".repo-language")
+
+        repoLanguage.forEach((span) => {
+          if (span.nextElementSibling.innerText === "JavaScript") {
+            span.style.backgroundColor = "#f1e05a"
+          } else if (span.nextElementSibling.innerText === "HTML") {
+            span.style.backgroundColor = "#e34c26"
+          } else if (span.nextElementSibling.innerText === "CSS") {
+            span.style.backgroundColor = "#664e88"
+          }
+        })
+      })
+
+      getPullRequests(forkedRepos)
+      drawChart(forkedRepos.length) // draw chart with the forkedRepos data
+    })
+}
+
+const getPullRequests = (allRepos) => {
+  //Get all the PRs for each project.
+  allRepos.forEach((repo) => {
+    fetch(
+      `https://api.github.com/repos/Technigo/${repo.name}/pulls?per_page=100`
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        const myPullRequest = data.find(
+          (pull) => pull.user.login === repo.owner.login
+        )
+        // Detect if we have pull request or not.
+        // If yes - call fetchCommits function
+        // If no - inform user that no pull request was yet done
+        if (myPullRequest) {
+          fetchCommits(myPullRequest.commits_url, repo.name)
+        } else {
+          document.getElementById(`commit-${repo.name}`).innerHTML =
+            "No pull request yet done :("
+        }
+
+        /* 1. Find only the PR that you made by comparing pull.user.login with repo.owner.login
+        2. Get the commits for each repo by using the commits_url as an argument to call another function
+        3. we can also get the comments for each PR by calling another function with the review_comments_url as argument */
+      })
+  })
+}
+
+// fetch the number of commits
+const fetchCommits = (myCommitsUrl, myRepoName) => {
+  fetch(myCommitsUrl)
+    .then((res) => res.json())
+    .then((data) => {
+      let commitMessage = data[0].commit.message
```

Comment:
Doesn't look like this variable got used, leftover from trying to fetch commit messages maybe?